src/saving/SaveHelper.cs

```diff
@@ -398,6 +427,9 @@ public static void ClearLastSaveTime()
     private static void InternalSaveHelper(SaveInformation.SaveType type, MainGameState gameState,
         Action<Save> copyInfoToSave, Func<Node> stateRoot, string? saveName = null)
     {
+        if (type == SaveInformation.SaveType.QuickSave && !AllowQuickSavingAndLoading)
+            return;
```

Comment:
One more tiny thing, could you add a print here warning the user why they couldn't save? That way if ever AllowQuickSavingAndLoading variable has a bug and isn't reset, we can see from the game logs why someone complains they can't save.

Reply:
To clarify it should be something like `GD.Print("Can't save due to quick save being currently suppressed");`